### PR TITLE
new designs for org members container

### PR DESCRIPTION
**Pull request by UnsungHero97**
Screen Shot 2020-09-09 at 6 30 26 PM

### CODE REVIEW

File: src/containers/org/members/OrgMembersContainer.js [hunk 1]
```diff
@@ -94,117 +54,20 @@ const OrgMembersContainer = ({ organization, organizationId } :Props) => {
...
-  const [addOrRemoveOrgMemberAction, setAddOrRemoveOrgMemberAction] = useState();
-  const [searchQuery, setSearchQuery] = useState();
-  const [targetMember, setTargetMember] = useState();
-  const [targetRole, setTargetRole] = useState<Role | void>();
+  const [selectedRole, setSelectedRole] = useState();
 
   const getOrganizationMembersRS :?RequestState = useRequestState([ORGANIZATIONS, GET_ORGANIZATION_MEMBERS]);
-  const searchAllUsersRS :?RequestState = useRequestState([USERS, SEARCH_ALL_USERS]);
 
   const isOwner :boolean = useSelector((s) => s.getIn([ORGANIZATIONS, IS_OWNER, organizationId]));
```
Comment (solomontang) [inline src/containers/org/members/OrgMembersContainer.js:61]: just curious, where does this isOwner get set? i would like to do something similar for CARE

**Review by solomontang — APPROVED** [review]

File: src/containers/org/components/AddRoleToOrgModal.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    const newRoleDescription = event.target.value || '';
+    setRoleDescription(newRoleDescription);
+  };
+
+  const handleOnClose = () => {
+    onClose();
+    // the timeout avoids rendering the modal with new state before the transition animation finishes
+    setTimeout(() => {
+      dispatch(resetRequestState([ADD_ROLE_TO_ORGANIZATION]));
+    }, 1000);
```
[the remark this answers is not in the document]
Comment (UnsungHero97) [reply src/containers/org/components/AddRoleToOrgModal.js:62]: I agree, it's an annoying bug.

File: src/containers/org/members/OrgMembersContainer.js (hunk 1 above)
Comment (UnsungHero97) [reply src/containers/org/members/OrgMembersContainer.js:61]: https://github.com/openlattice/lattice-orgs/blob/develop/src/containers/orgs/sagas/getOrganizationsAndAuthorizations.js

File: src/containers/orgs/OrgMemberContainer.js [hunk 3]
```diff
@@ -69,9 +68,9 @@ const OrgMemberContainer = ({ member, organization, organizationId } :Props) =>
   const orgPath = Routes.ORG.replace(Routes.ORG_ID_PARAM, organizationId);
 
   const userProfileLabel = getUserProfileLabel(member);
-  const userId = getUserId(member);
+  // const userId = getUserId(member);
```
Comment (giuliacampana) [inline src/containers/orgs/OrgMemberContainer.js:71]: do you need this comment anymore?

File: src/utils/PersonUtils.js [hunk 4]
```diff
@@ -122,48 +94,10 @@ function getUserProfile(user :any) :UserProfile {
...
-
 export {
-  filterUser,
   getPrincipalId,
-  getUserId,
+  // getUserId,
   getUserProfile,
   getUserProfileLabel,
-  sortByProfileLabel,
+  // sortByProfileLabel,
```
Comment (giuliacampana) [inline src/utils/PersonUtils.js:102]: do you need these comments anymore?

**Review by giuliacampana — APPROVED** [review]

File: src/containers/orgs/OrgMemberContainer.js (hunk 3 above)
Comment (UnsungHero97) [reply src/containers/orgs/OrgMemberContainer.js:71]: Nope. Actually, this component is not used anywhere. I've removed it in another branch.

File: src/utils/PersonUtils.js (hunk 4 above)
Comment (UnsungHero97) [reply src/utils/PersonUtils.js:102]: Nope. I've cleaned it up in another branch.

**Pull request merged**